test/Tests.cpp

```diff
@@ -146,70 +184,56 @@ NameValuePairMap CheckableValues()
     return Results;
 }
 
-void DoComparisonTest(const Mezzanine::NameValuePairMap& Expected, const Mezzanine::NameValuePairMap& Compiled)
+Mezzanine::String Stringify(const Mezzanine::NameValuePairMap& Mapping)
 {
-    cout << "Expected:\n" << Stringify(Expected)
-         << "\nCompiled in:\n" << Stringify(Compiled);
+    String Results;
+    for(Mezzanine::NameValuePair pair : Mapping)
+        { Results += "  " + pair.first + ": " + pair.second + "\n"; }
+    return Results;
+}
 
-    stringstream XmlContents;
-    XmlContents << "<testsuite tests=\"" << Expected.size() << "\">\n";
+String IntToString(Mezzanine::Int32 SomeInt)
```

Comment:
const?

Reply:
I didn't change this function other than maybe to move it. I will make this change if it is reasonable.